recipes/embree3/all/conanfile.py

```diff
@@ -0,0 +1,154 @@
+        target_name = "{0}-{1}".format("embree", self.version)
+        os.rename(target_name, self._source_subfolder)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+
+        # Configure CMake library build:
+        self._cmake.definitions["EMBREE_STATIC_LIB"] = not self.options.shared
+        self._cmake.definitions["BUILD_TESTING"] = False
+        self._cmake.definitions["EMBREE_TUTORIALS"] = False
+        self._cmake.definitions["EMBREE_GEOMETRY_CURVE"] = self.options.geometry_curve
+        self._cmake.definitions["EMBREE_GEOMETRY_GRID"] = self.options.geometry_grid
+        self._cmake.definitions["EMBREE_GEOMETRY_INSTANCE"] = self.options.geometry_instance
+        self._cmake.definitions["EMBREE_GEOMETRY_QUAD"] = self.options.geometry_quad
+        self._cmake.definitions["EMBREE_GEOMETRY_SUBDIVISION"] = self.options.geometry_subdivision
+        self._cmake.definitions["EMBREE_GEOMETRY_TRIANGLE"] = self.options.geometry_triangle
+        self._cmake.definitions["EMBREE_GEOMETRY_USER"] = self.options.geometry_user
+        self._cmake.definitions["EMBREE_RAY_PACKETS"] = self.options.ray_packets
+        self._cmake.definitions["EMBREE_RAY_MASK"] = self.options.ray_masking
+        self._cmake.definitions["EMBREE_BACKFACE_CULLING"] = self.options.backface_culling
+        self._cmake.definitions["EMBREE_IGNORE_INVALID_RAYS"] = self.options.ignore_invalid_rays
+        self._cmake.definitions["EMBREE_ISPC_SUPPORT"] = False
+        self._cmake.definitions["EMBREE_TASKING_SYSTEM"] = "INTERNAL"
+
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def build(self):
+        os.remove(os.path.join(self._source_subfolder, "common", "cmake", "FindTBB.cmake"))
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        cmake = self._configure_cmake()
+        cmake.install()
+        self.copy("LICENSE.txt", src=self._source_subfolder, dst="licenses")
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.rmdir(os.path.join(self.package_folder, 'lib', 'cmake'))
+        for command_file in glob.glob(os.path.join(self.package_folder, "*.command")):
+            os.remove(command_file)
+        for cmake_file in glob.glob(os.path.join(self.package_folder, "*.cmake")):
+            os.remove(cmake_file)
+
+        tools.rmdir(os.path.join(self.package_folder, "cmake"))
+
+        if self.options.shared:
+            for dll_pattern_to_remove in ["concrt*.dll", "msvcp*.dll", "vcruntime*.dll"]:
+                for dll_to_remove in glob.glob(os.path.join(self.package_folder, "bin", dll_pattern_to_remove)):
+                    os.remove(dll_to_remove)
+        else:
+            tools.rmdir(os.path.join(self.package_folder, "bin"))
+
+    def package_info(self):
+        self.cpp_info.libs = tools.collect_libs(self)
```

Comment:
From the documentation

> EMBREE_STATIC_LIB: Builds Embree as a static library (OFF by default). Further multiple static libraries are generated for the different ISAs selected (e.g. embree3.a, embree3_sse42.a, embree3_avx.a, embree3_avx2.a, embree3_avx512knl.a, embree3_avx512skx.a). You have to link these libraries in exactly this order of increasing ISA.

So you can't simply collect libs.

Reply:
Thank you very much. I can fix that later, or in a followup patch, when I'm available.